aiobotocore/endpoint.py

```diff
@@ -223,33 +230,38 @@ def __init__(self, host,
 
     async def _send_request(self, request_dict, operation_model):
         attempts = 1
+        meta_key = 'ResponseMetadata'
```

Comment:
the formatting here is explicit to match the version of this method in botocore, please ensure we match their method as much as possible to simplify changing for changes in new releases

Reply:
sounds good!